probeinterface/io.py

```diff
@@ -139,6 +139,58 @@ def read_prb(file):
     return probegroup
 
 
+def read_maxwell(file):
+    """
+    Read a maxwell file and return a ProbeGroup object.
+
+    Since Maxwell do not handle electrode shape then circle of 5um are put.
```

Comment:
Is it the case ?
Does it come from the hardware documentation ?

Reply:
They are not square in fact... They are rectangular, with a size of  9.3 × 5.45 μm2. Can we plot that?

Reply:
i found the solution myself, like a big boy

Reply:
un bon petit effectivement.